transformations/transformations/parallel_routine_dispatch.py

```diff
@@ -11,17 +11,48 @@
     FindNodes, nodes as ir
 )
 from loki.transform import Transformation
+from loki import (
+    FindVariables, DerivedType, SymbolAttributes,
+    Array, single_variable_declaration, Transformer,
+    BasicType
+)
+import pickle
+import os
 
 __all__ = ['ParallelRoutineDispatchTransformation']
 
 
 class ParallelRoutineDispatchTransformation(Transformation):
 
+    def __init__(self):
+        self.horizontal = [
+            "KLON", "YDCPG_OPTS%KLON", "YDGEOMETRY%YRDIM%NPROMA",
+            "KPROMA", "YDDIM%NPROMA", "NPROMA"
+    ]
+        #TODO : do smthg for opening field_index.pkl
+        with open(os.getcwd()+"/transformations/transformations/field_index.pkl", 'rb') as fp:
```

Comment:
Just as a remark for later: this map should be loaded/compiled/supplied in the head script, and then handed to the Transformation as an argument.